Documentation/ExtensionArchitecture/HowTo/UpdateExtensions/UpdateWizards/_SwitchableControllerActionUpgradeWizard.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace MyVendor\MyExtension\Upgrades;
+
+use Doctrine\DBAL\Exception;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Service\FlexFormService;
+use TYPO3\CMS\Core\Utility\DebugUtility;
+use TYPO3\CMS\Install\Attribute\UpgradeWizard;
+use TYPO3\CMS\Install\Updates\RepeatableInterface;
+use TYPO3\CMS\Install\Updates\UpgradeWizardInterface;
+
+#[UpgradeWizard('myExtension_switchableControllerActionUpgradeWizard')]
+final class SwitchableControllerActionUpgradeWizard implements UpgradeWizardInterface
+{
+    private const TABLE = 'tt_content';
+    private const PLUGIN = 'myextension_myplugin';
+    public function __construct(
+        private readonly ConnectionPool $connectionPool,
+        private readonly FlexFormService $flexFormService
```

Comment:
Unsure. Each lines will end with `,` now, right?

```suggestion
        private readonly FlexFormService $flexFormService,
```